FWCore/Services/plugins/SimpleMemoryCheck.cc

```diff
@@ -143,25 +167,19 @@ namespace edm {
 
       // Event summary statistics 				changeLog 1
       struct SignificantEvent {
-        int count;
-        double vsize;
-        double deltaVsize;
-        double rss;
-        double deltaRss;
-        bool monitorPssAndPrivate;
-        double privateSize;
-        double pss;
         edm::EventID event;
-        SignificantEvent()
-            : count(0),
-              vsize(0),
-              deltaVsize(0),
-              rss(0),
-              deltaRss(0),
-              monitorPssAndPrivate(false),
-              privateSize(0),
-              pss(0),
-              event() {}
+        double vsize = 0;
+        double deltaVsize = 0;
+        double rss = 0;
+        double deltaRss = 0;
+        double privateSize = 0;
+        double pss = 0;
+        double anonHugePages = 0;
+        JemallocInfo jemalloc;
```

Comment:
What if this was `std::optional<JemallocInfo>` which would allow you to remove `showJemallocInfo` variable and consolidate them into one idea.